setup.py

```diff
@@ -0,0 +1,30 @@
+"""Setup for aiohue."""
+from setuptools import find_packages, setup
+
+LONG_DESC = open("README.md").read()
+PACKAGES = find_packages(exclude=["tests", "tests.*"])
+REQUIREMENTS = list(val.strip() for val in open("requirements.txt"))
+MIN_PY_VERSION = "3.8"
+
+setup(
+    name="psutil-home-assistant",
```

Comment:
I don't really see how this code is home-assistant specific.

Wouldn't one of these be better?
- `psutil-wrapper`
- `psutil-copy`
- `psutil-multi`